Towerise/ClientApp/app/shared/Vector2.ts

```diff
@@ -19,6 +19,10 @@
         return new Vector2(this.x * multiplicationVector.x, this.y * multiplicationVector.y);
     }
 
+    public multByConst(multiplicationConstant: number) {
```

Comment:
i dont really think we need this function.  You can just make another parameter in mult, and then make then both optional.  If you include a number but not a Vector2 the function can multiply by that value.

Reply:
first off, your implementation of vector multiplication doesn't exist. 
Here's a scott link for ya
http://mathworld.wolfram.com/VectorMultiplication.html

multiplication by a scalar does exist though, so I would appreciate if you kept my clean, pure implementation of a mathematical basis intact


:P

Reply:
Well then change it's name to like scalar_multiply and change the input to a single digits.  I'd rather the function be right than me be right lol

Reply:
Alrighty, I fixed it